engine/redis_v2/queue_test.go

```diff
@@ -6,9 +6,9 @@ import (
 	"testing"
 	"time"
 
-	"github.com/go-redis/redis/v8"
-
 	"github.com/bitleak/lmstfy/engine"
+	"github.com/go-redis/redis/v8"
```

Comment:
Would you consider grouping the imports in order to be consistent with other files?

Reply:
For this point, I think we should group the imports order, but we don't require this before the CI integrate the import order check. Do you think this makes sense?

Reply:
sure